javascript/README.md

```diff
@@ -0,0 +1,11 @@
+### Example of integrating Salt Edge API with JavaScript.
+
+* After clone this code, in folder with files run any file server (like python -m SimpleHTTPServer) or another.
```

Comment:
To run this code, you need a simple http server that will server all the content. You can use `python -m SimpleHTTPServer`.